app/views/decidim/anonymous_codes/admin/code_groups/index.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="card">
+  <div class="card-divider">
+    <h2 class="card-title">
+      <%= t("code_groups.index.title", scope: "decidim.anonymous_codes.admin") %>
+      <a class="button tiny button--title new-access" href="<%= new_code_group_path %>"><%= t("code_groups.index.new_access_code_group_button", scope: "decidim.anonymous_codes.admin") %></a>
+    </h2>
+  </div>
+  <div class="card-section">
+    <% if @groups.any? %>
+      <div class="table-scroll">
+        <table class="table-list">
+          <thead>
+            <tr>
+              <th><%= t("code_groups.index.header_table_title", scope: "decidim.anonymous_codes.admin") %></th>
+              <th><%= t("code_groups.index.expires_at", scope: "decidim.anonymous_codes.admin") %></th>
+              <th><%= t("code_groups.index.active", scope: "decidim.anonymous_codes.admin") %></th>
+              <th><%= t("code_groups.index.num_of_tockens", scope: "decidim.anonymous_codes.admin") %></th>
+              <th><%= t("code_groups.index.max_reuses", scope: "decidim.anonymous_codes.admin") %></th>
+              <th></th>
+            </tr>
+          </thead>
+          <tbody>
+            <% groups.each do |group| %>
+              <tr>
+                <td><%= translated_attribute(group.title) %></td>
+                <td><%= l(group.expires_at, format: :decidim_short) %></td>
+                <td><%= t("booleans.#{group.active}") %></td>
+                <td><%= group.tokens_count %></td>
+                <td><%= group.max_reuses %></td>
+                <td class="table-list__actions">
+                  <%= icon_link_to "circle-x", code_group_path(group.id), t("actions.destroy", scope: "decidim.anonymous_codes.admin"), method: :delete, class: "action-icon--remove", data: { confirm: t("actions.confirm_destroy", scope: "decidim.anonymous_codes.admin") } %>
+                  <%= icon_link_to "pencil", edit_code_group_path(group.id), t("actions.edit", scope: "decidim.anonymous_codes.admin"), class: "action-icon--edit" %>
+                  <%= icon_link_to "eye", resource_path(group), t("actions.preview", scope: "decidim.anonymous_codes.admin"), class: "action-icon--preview#{group&.resource} ? '':' invisible'", target: :_blank %>
```

Comment:
```suggestion
                  <%= icon_link_to "eye", resource_path(group), t("actions.preview", scope: "decidim.anonymous_codes.admin"), class: "action-icon--preview#{group&.resource ? '':' invisible'}", target: :_blank %>
```
All the ruby instruction mus go inside the interpolation block

Reply:
Done